src/azul/service/manifest_service.py

```diff
@@ -878,7 +884,11 @@ def _get_related_rows(self, doc: dict, row: dict) -> Iterable[dict]:
         file_ = one(doc['contents']['files'])
         for related in file_['related_files']:
             new_row = row.copy()
-            new_row.update({'file_' + k: v for k, v in related.items()})
+            for field_name, field_value in related.items():
```

Comment:
The fact that this does not require any changes to the corresponding tests, has me worried. Let's factor the compact manifest changes into a separate ticket and make sure we cover this code in tests. Please determine who would have been responsible for adding the tests and assign the ticket to them.